src/main/java/com/example/WorldOfAnimals/controllers/AnimalsController.java

```diff
@@ -0,0 +1,31 @@
+package com.example.WorldOfAnimals.controllers;
+
+
+import com.example.WorldOfAnimals.models.AnimalBehavior;
+import com.example.WorldOfAnimals.models.AnimalsGroup;
+import com.example.WorldOfAnimals.services.AnimalsService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping
```

Comment:
Maybe prefix animals need set to RequestMapping ? 
@RequestMapping("/animals")

Reply:
done